app/views/films/index.html.erb

```diff
@@ -33,46 +30,53 @@
 <% end %>
 <br>
 
+
 <caption><h1>Movies From Database</h1></caption>
+<div class="table-container">
 <table class="table table-bordered">
   <thead>
   <tr>
-    <th>#</th>
-    <th>Rating</th>
-    <th>Genre</th>
-    <th>Year</th>
-    <th>Title</th>
-    <th>View Details</th>
+    <th scope="col">No</th>
+    <th scope="col">Rating</th>
+    <th scope="col">Genre</th>
+    <th scope="col">Yts_Id</th>
+    <th scope="col">Api_Id</th>
+    <th scope="col">Year</th>
+    <th scope="col">Title</th>
+    <th scope="col">View Details</th>
     <th scope="col">Image</th>
   </tr>
   </thead>
   <tbody>
   <% if @films.empty? %>
-    <tr>
-      <td colspan="6">No movies found in the database.</td>
-    </tr>
   <% else %>
     <% @films.each_with_index do |film, index| %>
```

Comment:
why you need to do here `.each_with_index` why you need to use index in the code.
remove this.